packages/fxa-settings/src/components/App/HeaderContinueTo/index.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+
+export type HeaderContinueToProps = {
+  prefixDataTestId: string;
+  prefixFtlId: string;
+  headingText: string;
+  serviceName?: string;
+};
+
+const HeaderContinueTo = ({
+  prefixDataTestId,
+  prefixFtlId,
+  headingText,
+  serviceName,
+}: HeaderContinueToProps & RouteComponentProps) => {
+  return (
+    <header data-testid={`${prefixDataTestId}-header`} className="mb-4">
+      <Localized id={`${prefixFtlId}-heading`}>
+        <h1 className="text-xl mb-2">{headingText}</h1>
+      </Localized>
+      {!serviceName && (
+        <Localized id="header-continue-to-default-subheading">
+          <h2>{`to continue to account settings`}</h2>
+        </Localized>
+      )}
+      {serviceName && (
+        <Localized id="header-continue-to-custom-subheading">
+          <h2>{`to continue to ${serviceName}`}</h2>
+        </Localized>
```

Comment:
We usually prefer ternaries here, search for `? (` in our TSX and you'll see examples.

Also, the 2nd part of the heading shouldn't be an `h2` because it's a single sentence a screenreader would read separately and the DOM hierarchy will look a little funky, it should be a `span`.